doc/admin-guide/plugins/statichit.en.rst

```diff
@@ -0,0 +1,106 @@
+.. _admin-plugins-statichit:
+.. include:: ../../common.defs
+
+Static Hit Plugin
+*****************
+
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+
+This is a simple plugin to serve static content from the proxy's local filesystem. It shares some
+of the same functionality as the `healthchecks` plugin, but is a remap plugin (thereby making it
+reloadable). It does not use fsnotify for watching the source files.
```

Comment:
Is this last sentence meant to imply that source file changes will need a reload to begin serving? If so, perhaps say that explicitly? like
"It does not use fsnotify for watching the source files. New content will be served after |TS| reloads."